### ISSUE TITLE
Address Aqua alert for openssl, libcrypto3, and libssl3

### ISSUE THREAD

**Issue by yoomlam** [department-of-veterans-affairs/abd-vro#1495]
To address a [Dockerfile build error](https://dsva.slack.com/archives/C04CA47HV96/p1682014641521959?thread_ts=1682013971.118889&cid=C04CA47HV96) (and unblock creating a release for 4/24 VRO deployment), I removed the explicit versions for the apk packages: openssl, libcrypto3, and libssl3, and thereby undoing @greene858's PR #1382. However this results in Aqua alerts. This ticket should address those Aqua alerts without causing the Dockerfile build to fail -- it may not fail immediately; it could fail a month later (maybe because versions in the `apk` world have been updated?).

**Comment by msnwatson** [comment]
Could be wrong but isn't this done? @greene858 @yoomlam @tcraghu

**Comment by yoomlam** [comment]
I believe Tom worked on this -- check merged PRs.

@greene858 We had talked about bubbling up acknowledged SecRel alerts so they get included in sprints and don't pile up and reappear when we need to do a prod deployment. What's the plan for this?

**Comment by msnwatson** [comment]
Yeah, seems like this was addressed either through some base image version upgrade or something because the alerts have stopped showing up in aqua runs.

When we get @greene858 's input we can decide if we want to spin out a new ticket that gets at this "bubbling up" process. I'll leave this open for that discussion, but we can close this ticket technically.

**Comment by greene858** [comment]
This SecRel issue has been addressed
